sdk/core/switcher/entry.S

```diff
@@ -18,6 +18,15 @@
  */
 #define CSR_MSHWMB 0xbc2
 
+/**
+ * The special capability register for the fast zeroing control.
+ * Writing a capability here starts zeroing from the address of this register
+ * down to the base.
+ *
+ * TODO: This is a made up number, it might need to be a different one.
+ */
+#define SCR_ZTOP 0x1c
```

Comment:
0x1c is `mtcc`.  Ibex is using `0x1b` for this.  That was an exciting thing to discover in testing.